test/e2e/data/infrastructure-cloudstack/v1beta2/cluster-template-invalid-ip/cloudstack-cluster.yaml

```
@@ -0,0 +1,17 @@
+piVersion: infrastructure.cluster.x-k8s.io/v1beta2
```

```suggestion
apiVersion: infrastructure.cluster.x-k8s.io/v1beta2
```

Oops. Kustomize can't build the final template without the `a` : )

Happy to merge after the fixup.